db/src/Pos/DB/Update/Poll/DBPoll.hs

```diff
@@ -1,5 +1,7 @@
 {-# LANGUAGE TypeFamilies #-}
 
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
```

Comment:
Is that really needed? I don't see any typeclass instances added to this file.

Reply:
It's for `instance (...) MonadDBPollRead DBPoll`, because `DBPoll` is a type alias and not a newtype.